clusterDeployer.py

```diff
@@ -191,6 +191,10 @@ def teardown(self):
         print(lh.run("virsh net-start default"))
         print(lh.run("systemctl restart libvirtd"))
 
+      pool_name = self.images_pool_name()
+      print(lh.run(f"virsh pool-destroy {pool_name}"))
+      print(lh.run(f"virsh pool-undefine {pool_name}"))
```

Comment:
Again, this is somewhat of an issue with the current default pool name. Can you change it so that each cluster has a unique pool name? Your previous patches already allow the "guest_images" pool to be moved, but I think we should rename the pool itself to something that is specific to the cluster, i.e. `f"{cluster_name}_images"`

Reply:
> Again, this is somewhat of an issue with the current default pool name. Can you change it so that each cluster has a unique pool name? Your previous patches already allow the "guest_images" pool to be moved, but I think we should rename the pool itself to something that is specific to the cluster, i.e. `f"{cluster_name}_images"`.

?!? with the previous patch the pool name becomes exactly <cluster name>"_guest_images":

```
def images_pool_name(self):
      return self._cc["name"]+"_guest_images"

```

If in some place the new code does not use the above construct, as in the next patch, that is a bug. But I don't see other relevant chunks ?!?

Reply:
You're right. I confused pool name with path. Thanks for pointing that out.